test/fixtures/versions.yml

```diff
@@ -139,3 +139,37 @@ page1_v5:
     site_id: 1
     page_id: 11
     version_id: 115
+
+page3_v1:
+  page: home_page
```

Comment:
Also, please use a different actual `page` from the one we are using for `page1_v*` fixtures when making a fixture named `page3_v*`. Same for the other new fixtures here.